data/json/scores/achievements.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "id": "achievement_kill_zombie",
+    "type": "achievement",
+    "description": "One down, billions to go\u2026",
+    "requirements": [ { "event_statistic": "num_avatar_zombie_kills", "is": ">=", "target": 1 } ]
+  },
+  {
+    "id": "achievement_swim_short_distance",
+    "type": "achievement",
+    "description": "A Swimming Merit Badge",
```

Comment:
Yeah, I thought the same but only had your single example so far. I don't have a real strong opinion either--I just picked the title case because that's the convention that I see everywhere. Your system, you decide and I'll update 😄 .

Reply:
I've looked at Steam and it appears that you're right.  Almost everyone seems to be capitalizing every word.  But Factorio doesn't, and that's clearly the most important game to imitate ;).

Ultimately I think it comes down to what style of name we're using.  If they were mostly one to three word names, then maybe All Initial Caps would work, but we seem to be leaning towards longer names, where I think it starts to look silly.  So I'm voting for not doing that.   Just use regular sentence case.